.changeset/chilled-scissors-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/example-ecommerce': patch
```

Comment:
Thinking about #4462, I don't think this sort of refactor is worth a changeset. It's basically duplicating the commit history for the example into the changelog. But maybe that's more accessible for people who want to follow along? I could probably be convinced if you have strong views, but I haven't been adding changesets for similar things and we should be consistent.

Reply:
Experimented with this in #4463